src/main/groovy/com/morpheusdata/scvmm/ScvmmProvisionProvider.groovy

```diff
@@ -867,7 +843,33 @@ class ScvmmProvisionProvider extends AbstractProvisionProvider implements Worklo
             log.error("runWorkload error:${e}", e)
             provisionResponse.setError(e.message)
             return new ServiceResponse(success: false, msg: e.message, error: e.message, data: provisionResponse)
-        }
+        } finally {
+			// Always power on the parent VM in case of a clone operation
+			if (scvmmOpts.cloneVMId && scvmmOpts.cloneContainerId) {
```

Comment:
-Log the exception that may happen during startserver or cdrom operations.
- Return an warning response for all operations, with generic messages - Unable to unset cdrom, please manually perform ...
- Refactor into a new method.